Woof/WoofTests/ViewModel/SitterProfileViewModelTests/SitterProfileViewModelSaveTests.swift

```diff
@@ -127,4 +129,52 @@ final class SitterProfileViewModelSaveTests: XCTestCase {
         // Then
         XCTAssertEqual(requestCount, 1)
     }
+
+    func testFirstSaveChangesSitterIsSet() async {
+        // Given
+        MockURLProtocol.requestHandler = { request in
+            let response = try XCTUnwrap(
+                HTTPURLResponse(
+                    url: XCTUnwrap(request.url),
+                    statusCode: 200,
+                    httpVersion: nil,
+                    headerFields: nil
+                )
+            )
+            return (response, nil)
+        }
+
+        // When
+        let sitterIsSetBeforeSave = viewModel.sitterIsSet
+        await viewModel.save()
+        let sitterIsSetAfterSave = viewModel.sitterIsSet
+
+        // Then
+        XCTAssertFalse(sitterIsSetBeforeSave)
+        XCTAssertTrue(sitterIsSetAfterSave)
+    }
+
+    func testSecondSaveDoesntChangeSitterIsSet() async {
```

Comment:
nit: The test name is not clear to me.
```suggestion

    func testSitterIsSetIsNotChangedWhenSaveCalledTwice() async {
```

Reply:
Changed with 7f84b49